rwgame/main.cpp

```diff
@@ -1,11 +1,10 @@
 #include <iostream>
 #include "RWGame.hpp"
 #include "SDL.h"
-
 #include <core/Logger.hpp>
 
 int main(int argc, char* argv[]) {
-    // Initialise Logging before anything else happens
+    // Initialize Logging before anything else happens
```

Comment:
We don't have a proscribed spelling, so I don't think it's worth correcting regional spelling.